CMakeLists.txt

```diff
@@ -1,85 +1,47 @@
-cmake_minimum_required(VERSION 3.5)
-project(node-libuiohook)
+cmake_minimum_required(VERSION 3.0)
+project(node_libuiohook)
```

Comment:
What is the reason behind changing the project name to have an underscore instead of a dash?